### DISCUSSION TITLE
Lower and uppercase trino

### DISCUSSION THREAD

**Discussion by pcontrerasnun** [great-expectations/great_expectations#5834] [Support]
Hi there!

I was trying to run some expectations on ClickHouse, but since it is not fully supported I was having some issues. As a way around I tried using Trino. The thing is I defined my expectations using ClickHouse tables where column names are uppercase. On Trino my columns are lowercase so when I run validations Trino says it can't find the columns.

Is there a way to do a not case sensitive batch_request?

Thanks in advance!

**Comment by kyleaton** [comment]
Hi @pcontrerasnun, 

We have a similar issue open here: https://github.com/great-expectations/great_expectations/issues/5535

Would you mind chiming in there and following along? 

Thanks, 
Kyle
